### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by rnurgaliyev**
Update automation code so it will update mongos to latest when it runs. Now, if there is any version installed (3.2) it will not be updated to latest (3.4) when one runs role.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 - name: main | install mongos package
   apt:
     name="mongodb-org-mongos"
-    state="present"
+    state="latest"
```
Comment (olvlvl) [inline tasks/main.yml:30]: Shouldn't we use a specific number to match what we are using currently, which is `3.4.6`?

Comment (rnurgaliyev) [reply tasks/main.yml:30]: No apt does not work like that

Comment (olvlvl) [reply tasks/main.yml:30]: You can specify the version with `sudo apt-get install package=version`. But maybe that's not a good idea?

Comment (rnurgaliyev) [reply tasks/main.yml:30]: Not many options are usually there, however I've just checked and mongo repository indeed has lots of different versions available.

**Pull request closed without merging**